### PR TITLE
Declaration and consistency improvements.

### PR DESCRIPTION
**Pull request by TeMPOraL**
Fixes an invalid type declaration and resolves issue #2.

### CODE REVIEW

File: string-pokemonize.lisp [hunk 1]
```diff
@@ -20,7 +20,7 @@
                        (DeBuG 1)
                        (sAfEtY 1)))
     (FlEt ((pOkEmOnIzE (cHaR)
-             (dEcLaRe (ExTeNdEd-cHaR ChAr))
+             (dEcLaRe (ChArAcTeR ChAr))
```
Comment (phoe) [inline string-pokemonize.lisp:23]: `(dEcLaRe (ChArAcTeR ChAr))`
should be
`(dEcLaRe (ChArAcTeR cHaR))`
which in turn forces a case change for all characters after this line.

In order to merge this, I need the case of the code to be consistent. Please format the file using the pokemonize case and reupload it.

File: string-pokemonize.asd [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-  :author "Michał \"phoe\" Herda <phoe@openmailbox.org>"
-  :license "MIT license"
-  :serial t
-  :components ((:file "string-pokemonize")))
+(AsDf:dEfSyStEm #:sTrInG-PoKeMoNiZe
+  :DeScRiPtIoN "tHe mIsSiNg cOmMoN LiSp sTrInG FuNcTiOn"
+  :aUtHoR "mIcHaŁ \"PhOe\" hErDa <PhOe@oPeNmAiLbOx.oRg>"
+  :LiCeNsE "mIt lIcEnSe"
+  :sErIaL T
+  :cOmPoNeNtS ((:fIlE "sTrInG-PoKeMoNiZe")))
```
Comment (phoe) [inline string-pokemonize.asd]: We have no file called `sTrInG-PoKeMoNiZe.lisp` - we have case-sensitive filesystems.
Have you built the project before submitting this PR?

**Review by phoe — CHANGES_REQUESTED** [review]
Changes and verification requested.

File: string-pokemonize.lisp (hunk 1 above)
Comment (TeMPOraL) [reply string-pokemonize.lisp:23]: It was formatted using Pokemonize. You're missing a space between #\R and #\C.

File: string-pokemonize.asd (hunk 2 above)
Comment (TeMPOraL) [reply string-pokemonize.asd]: No, of course I didn't. Why would I?

Comment (TeMPOraL) [reply string-pokemonize.asd]: Fixed.

File: string-pokemonize.lisp (hunk 1 above)
Comment (phoe) [reply string-pokemonize.lisp:23]: You're correct - then it means that we have a bug in the implementation somewhere, the case is flipped even when non-cased characters are read. I'll look into it later and merge it for now.

**Pull request merged**